### PR TITLE
adding hello endpoint

### PR DESCRIPTION
**Pull request by ajaypattanashetty**
### Summary by OpenAI

New Feature: Added a `/hello` endpoint that returns "Hello world!" when accessed. Also added an acceptance test for the `HelloController` endpoint to ensure proper functionality. 

> "Hello world!" we say,
> With a new endpoint today.
> Acceptance test in place,
> Bugs and vulnerabilities we'll erase.

### CODE REVIEW

File: src/__tests__/acceptance/hello.controller.acceptance.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    await app.stop();
+  });
+
+  it('invokes GET /hello', async () => {
+    const res = await client.get('/hello').expect(200);
+    console.log(res.body);
+    // expect(res.body.toString()).to.equal('Hello world!');
+    expect(res.text).to.be.equal('Hello world!');
+  });
+});
```
[the remark this answers is not in the document]
Comment (ajaypattanashetty) [reply src/__tests__/acceptance/hello.controller.acceptance.ts:27]: @openai add more test cases to cover edge cases and error scenarios
